app/src/main/java/com/cornellappdev/score/screen/GameDetailsScreen.kt

```diff
@@ -0,0 +1,108 @@
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.LocationOn
+import androidx.compose.material3.Divider
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.cornellappdev.score.R
+import com.cornellappdev.score.model.ScoreEvent
+import com.cornellappdev.score.model.Team
+
+@Composable
+fun GameDetailsScreen(scoreEvents: List<ScoreEvent>, onArrowClick: () -> Unit) {
+    Column(
```

Comment:
I assume this is still missing some stuff before it's the full screen? Add a TODO if you're not planning to finish that in this PR!